modules/javafx.graphics/src/test/java/test/com/sun/javafx/pgstub/StubFontLoader.java

```diff
@@ -47,123 +47,127 @@ public class StubFontLoader extends FontLoader {
     public void loadFont(Font font) {
         StubFont nativeFont = new StubFont();
         nativeFont.font = font;
+
         String name = font.getName().trim().toLowerCase(Locale.ROOT);
-        if (name.equals("system") || name.equals("system regular")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "System", "Regular");
-        } else if (name.equals("amble regular")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble", "Regular");
-        } else if (name.equals("amble bold")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble", "Bold");
-        } else if (name.equals("amble italic")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble", "Italic");
-        } else if (name.equals("amble bold italic")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble",
-                    "Bold Italic");
-        } else if (name.equals("amble condensed")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble Cn", "Regular");
-        } else if (name.equals("amble bold condensed")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble Cn", "Bold");
-        } else if (name.equals("amble condensed italic")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble Cn", "Italic");
-        } else if (name.equals("amble bold condensed italic")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble Cn",
-                    "Bold Italic");
-        } else if (name.equals("amble light")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble Lt", "Regular");
-        } else if (name.equals("amble light italic")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble Lt", "Italic");
-        } else if (name.equals("amble light condensed")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble LtCn",
-                    "Regular");
-        } else if (name.equals("amble light condensed italic")) {
-            FontHelper.setNativeFont(font, nativeFont, font.getName(), "Amble LtCn",
-                    "Italic");
+        switch (name) {
+            case "system regular" -> FontHelper.setNativeFont(font, nativeFont, font.getName(), "System", "Regular");
+            case "system bold" -> FontHelper.setNativeFont(font, nativeFont, font.getName(), "System", "Bold");
+            case "system italic" -> FontHelper.setNativeFont(font, nativeFont, font.getName(), "System", "Italic");
+            case "system bold italic" ->
+                    FontHelper.setNativeFont(font, nativeFont, font.getName(), "System", "Bold Italic");
```

Comment:
minor suggestions:
font -> f
nativeFont -> nf
this will nicely fit each case on a single line (and reduce clutter)